.github/workflows/ci.yml

```
@@ -274,7 +274,7 @@ jobs:
         uses: conda-incubator/setup-miniconda@v2
         with:
           activate-environment: modin_on_omnisci
-          python-version: 3.7.8
+          python-version: 3.7
```

The minimum version for pandas is 3.7.1
```suggestion
          python-version: 3.7.1
```

---

When we define python version like that: `python-version: 3.7` the latest version available on the channel is installed. At the moment it is `3.7.10`.

---

Sounds good